src/models/db/transfers.js

```diff
@@ -31,6 +36,11 @@ function convertFromPersistent (data) {
   if (data.rejected_at) {
     data.rejected_at = new Date(data.rejected_at)
   }
+  if (!isNil(data.rejection_reason_id)) {
```

Comment:
Is this just a safety check? Or do some dbs return `undefined` and others return `null`?